### DISCUSSION TITLE
https://blog.ghamza.dev/posts/computer-compression/

### DISCUSSION THREAD

**Discussion by giscus** [ghamza-blog/ghamza-blog.github.io#3] [General]
# https://blog.ghamza.dev/posts/computer-compression/

The science behind decreasing file sizes.

https://blog.ghamza.dev/posts/computer-compression/

**Comment by Ghamza-Jd** [comment]
Another example came on my mind.
If you want to pass a paper to your friend in school smoothly without making the teacher notice, you have to fold it as much as possible so that the transfer process is as fast as possible. Similarly in computers, we compress files before storing them on the cloud, so the upload and the download of the files are minimal.
In fact, compression runs the internet, when you open a website you receive a `gzip` file and your browser decompress it and encode it to a useful data and style for you to see.
